### PR TITLE
Hotfix v0.14

### PR DESCRIPTION
**Pull request by Alan-Cha**
Related to #1458

### CODE REVIEW

**Comment by Alan-Cha** [comment]
I did not change the version in the `testdata` files. I'm not sure if I should but I tested one or two, and they still work with `v0.13`.

**Comment by Alan-Cha** [comment]
* `Additional Helm chart linting / ...`: failing due to preexisting issue #1460 (4 failed tests)
* `Test experiment charts using testdata charts / unit-test`: failing due to `0.14` not being in iter8 chart hub (should be resolved after this PR) (1 failed test)
* `Test experiment charts using local charts / ...`: failing due to no `0.14` Docker image (should be resolved after making a release after this PR) (8 failed tests)
* `Test kustomize experiments / ...`: failing due to no `0.14` Docker image (should be resolved after making a release after this PR) (4 failed tests)

Total 17 failed checks

File: Dockerfile.dev [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# Small linux image with iter8 binary
+# Small Linux image with Iter8 binary
```
Comment (sriumcp) [inline Dockerfile.dev]: Could we please remove the `Dockerfile.dev` file altogether? We can (later, in a separate PR), create an *example* of how to create your own Dockerfile with Iter8 in it (for local development purposes).

We really don't want to maintain this. For e.g., this is starting to diverge now from the main Dockerfile.

**Review by sriumcp — CHANGES_REQUESTED** [review]
Minor change requested.

File: Dockerfile.dev (hunk 1 above)
Comment (Alan-Cha) [reply Dockerfile.dev]: Okay, sure

Comment (Alan-Cha) [reply Dockerfile.dev]: Done

**Review by sriumcp — APPROVED** [review]
lgtm

**Comment by Alan-Cha** [comment]
Fewer failing tests, 15, than before, 17, but that seems to be because failed tests prematurely stop the workflow and the number of failed tests is non-deterministic. `Test kustomize experiments` has fewer failed tests this time. Everything still seems fine. Feel free to merge.

**Pull request merged**